apps/package.json

```diff
@@ -170,6 +170,7 @@
   "dependencies": {
     "filesaver.js": "0.2.0",
     "jszip": "3.0.0",
-    "query-string": "4.1.0"
+    "query-string": "4.1.0",
+    "react-addons-update": "^15.3.0"
```

Comment:
This should be in dev-dependencies instead of dependencies, but I think we can avoid using it entirely.